schools/aggiecorps.yml

```diff
@@ -8,6 +8,10 @@ theme:
   background: 'https://static.gradebook.app/sYbR9JXKTI/aggiecorps.jpg'
 gpa: true
 partner: true
```

Comment:
Do we still have areas where `partner` is used?

Reply:
Yes, this is an API change so it needs to be done incrementally

Reply:
Sounds good, as long as we note it somewhere to clean it up later 👍